app/controllers/idv/review_controller.rb

```diff
@@ -18,8 +19,8 @@ def confirm_current_password
 
       analytics.idv_review_complete(
         success: false,
-        fraud_review_pending: current_user.fraud_review_pending?,
-        fraud_rejection: current_user.fraud_rejection?,
+        fraud_review_pending: fraud_review_pending?,
+        fraud_rejection: fraud_rejection?,
```

Comment:
This is tangential to this PR, but we should definitely not be in this controller if `fraud_rejection?` returns `true`. Also we don't have a feature test that checks that a user who has been rejected for fraud is blocked for IdV.